dlk/python/dlk/core/optimizer.py

```diff
@@ -364,12 +365,72 @@ def pass_pack_weights(graph: Graph) -> None:
         data = packer.run(op_data.astype(np.float32), weight_quantizer.dimension)
 
         # Create the new constant with the quantized weights
+        def transpose_kernels(kernel_data: np.ndarray,
```

Comment:
I prefer to avoid nested function except closure. Could you change the function to none nested ?

See also 
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#264-decision

Reply:
I do not have confidence, but I think that only the function of pass_ should be present at the top level of optimizer.

Reply:
But, OK. I will fix it.

Reply:
In python manner, adding `_` prefix is local function.
https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces
Thus, recommend name is `def _transpose_kernels()`, and don't need care it is top level.